README.md

```diff
@@ -94,19 +106,25 @@ terraform apply --auto-approve
 Optional resource creations are disabled by default. To enable the creation of a specific resource named `X`, navigate to [config/terraform.tfvars](./config/terraform.tfvars) and update the value of `enable_X` to `true` before applying the tempate.
 
 Created resources (if all enabled):
-* EIP allocated for the load balancer created by Ingress-NGINX
-* Karpenter provisioner, the node template and the SQS interruption queue
-* Metrics Server along with the Kubernetes Dashboard and the read-only user
-* MSK cluster featuring Kafka brokers and zookeepers
-* RDS instance running managementportal, appserver and rest_sources_auth databases
-* Route53 zone and records accompanied by IRSAs for external DNS and Cert Manager
-* S3 buckets for intermediate-output-storage, output-storage and velero-backups
-* SES SMTP endpoint
-* CloudWatch event rules and targets
-* Essential IAM policies, roles, users for aforementioned resources
+
+- EIP allocated for the load balancer created by Ingress-NGINX
+- Karpenter provisioner, the node template and the SQS interruption queue
+- Metrics Server along with the Kubernetes Dashboard and the read-only user
+- MSK cluster featuring Kafka brokers and zookeepers
+- RDS instance running managementportal, appserver and rest_sources_auth databases
+- Route53 zone and records accompanied by IRSAs for external DNS and Cert Manager
+- S3 buckets for intermediate-output-storage, output-storage and velero-backups
+- SES SMTP endpoint
+- CloudWatch event rules and targets
+- Essential IAM policies, roles, users for aforementioned resources
+
+## Contributing
+
+Make sure to install [terraform-docs](https://github.com/terraform-docs/terraform-docs) and run `make prepare` before making a commit to make sure the documentation is up to date and the code is valid.
```

Comment:
@baixiac  There is a brief description about it here

Reply:
Oh sorry, it slipped through somehow. Sounds like it would be a good use case for the git pre-commit hook?

Reply:
Yes it would be good for that but it needs extra configuration and documentation that I think we can do at a later time.